openapi/openapi/src/test/scala/endpoints4s/openapi/EndpointsTest.scala

```diff
@@ -422,3 +502,51 @@ object Fixtures
     )
 
 }
+
+trait MyFixture extends algebra.Endpoints with algebra.JsonEntitiesFromSchemas {
```

Comment:
Is this used?

Reply:
Oupsy, That was just a quick test I did that wasn't intended to be checked-in. I'll clear that out...